components/Layout.tsx

```diff
@@ -329,128 +329,6 @@ const Footer = () => (
   </footer>
 );
 
-const OpenJS = () => (
```

Comment:
Why pushing this change? please remove from the PR anything that is unrelated to this feature.